lib/bootloader_setup.pm

```diff
@@ -724,7 +724,9 @@ sub specific_bootmenu_params {
         return " @params ";
     }
 
-    type_string_very_slow " @params " if @params;
+    if (@params) {
+        type_string(" @params ", max_interval => get_var('TYPE_BOOT_PARAMS_FAST') ? $utils::VERY_SLOW_TYPING_SPEED : undef);
```

Comment:
Looks good from my point of view but, maybe extract this and all of the other calls to a routine inside this library, say `type parameters`? would make the code a bit cleaner